practice/hello.c

```diff
@@ -8,5 +8,5 @@
 
 #include <stdio.h>
 int main () {
-	printf ("Hello,World");
+	printf ("Hello,World\n");
 }
```

Comment:
You have declared main as returning an int. But nothing is being returned.
While it is unimportant in this case, you should develop the practice of using return codes according to the signature of the function.